### ISSUE TITLE
What's up with cylinder numbers

### ISSUE THREAD

**Issue by rusefillc** [rusefi/hellen154hyundai-issues#2]
Why "1 or 4"?

![image](https://user-images.githubusercontent.com/48498823/127369102-594494e6-a8aa-4cad-adb4-4831c17f540b.png)

**Comment by ElDominio** [comment]
Spark outputs change if it is immobilizer equipped, they switch cylinder numbers when an immobilizer is pesent. It changes the phase by 360 degrees (so, if the trigger angle is 60, the new sync angle would be around 400)

**Comment by rusefillc** [comment]
Hold on, WHAT? why?

**Comment by mck1117** [comment]
wait, what?

**Comment by ElDominio** [comment]
uh I dunno ask Hyundai and their engineering department LOL

**Comment by mck1117** [comment]
wait, are we sure it isn't just wasted spark? or temporarily wasted spark during sync, then transition to sequential?

are there other outputs labeled the inverse of those? (like `cyl 1 (or cyl 4)`)

**Comment by mck1117** [comment]
indeed it does - all 4 are present:

![image](https://user-images.githubusercontent.com/568254/127375073-69132149-ea16-4473-ba96-bef293bb48f2.png)

**Comment by mck1117** [comment]
note on schematic about immo here

![image](https://user-images.githubusercontent.com/568254/127375110-7bbe2816-dc6f-4107-8644-69db22e561dc.png)

**Comment by ElDominio** [comment]
Yeah like I said heh. IDK why they do it, but they did do it. It can be implemented with a software switch in the firing order and that's all
